### PR TITLE
add strongly collectionwise normal

### PR DESCRIPTION
**Pull request by StevenClontz**
See #994.

### CODE REVIEW

**Comment by prabau** [comment]
I am a little confused by the description of the space S207.
Isn't that the same as the product of a discrete space of size $\omega_1$ with a Fortissimo space of size $\omega_1$?

Or another way of seeing this, isn't it the same as the topological sum of $\omega_1$ (or $\aleph_1$) many copies of the Fortissimo space on $\omega_1$ (= one-point Lindelofication of a discrete space of size $\omega_1$) ?

**Comment by StevenClontz** [comment]
The trick is that we need the usual product topology neighborhoods (both vertical and horizontal) for points in $\omega_1\times\{\omega_1\}$.

**Comment by StevenClontz** [comment]
Adding that characterization explicitly couldn't hurt

**Comment by prabau** [comment]
I see.  Tricky!

File: properties/P000207.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+refs:
+  - zb: "0046.16403"
+    name: Sur une problème de M. Dieudonné (Cohen)
+---
+
+For each neighborhood $U$ of the diagonal $\Delta=\{(x,x)\mid x\in X\}$
+in $X\times X$, there is a neighborhood $V$ of the diagonal such that
+$V\circ V\subseteq U$.
+
+Studied by Cohen in {{zb:0046.16403}}, where it was shown to be
```
Comment (prabau) [inline properties/P000207.md:15]: Also add the direct link to the article, as was done for S207.

**Comment by prabau** [comment]
I think it would be useful to add https://doi.org/10.1090/S0002-9939-1981-0630058-4  to the refs for P207.  At least it mentions both terminologies.

**Comment by prabau** [comment]
One question.  The theorems provided already imply that both hereditarily connected spaces and spaces with a focal point are strongly collectionwise normal.  What both of these have in common is that they are ultraconnected.

Would an ultraconnected space in general be strongly collectionwise normal?

**Comment by StevenClontz** [comment]
I should have mentioned this - I spent a very small amount of time trying to prove ultraconnected spaces are strongly collectionwise normal, but I was unsuccessful. (Is it true that the only neighborhood of the diagonal in an ultraconnected space is the whole square?)

**Comment by prabau** [comment]
Yeah, that's the question I was wondering about.  My guess would be no, but no obvious example came to mind.

**Comment by david20000813** [comment]
Ultraconnected spaces are indeed divisible, for the same reason - any neighborhood of the diagonal is the entire space. I've posted an answer to Steven's question on MSE showing this.

**Comment by StevenClontz** [comment]
It turns out that this is even a characterization for ultraconnected spaces! https://math.stackexchange.com/questions/5006424/are-ultraconnected-spaces-divisible

**Comment by StevenClontz** [comment]
Thanks @david20000813 , marking as draft as I improve things with this new result

**Comment by Moniker1998** [comment]
As mentioned in #994 the example of an uncountable $\Sigma$-product of $\omega$ is an example of a strongly collectionwise normal space that's not fully normal.
This should be in Corson's *Normality in subsets of product spaces*

**Comment by prabau** [comment]
That would be good for a separate PR.

**Review by david20000813 — APPROVED** [review]

**Comment by prabau** [comment]
@StevenClontz There seems to be a CRLF problem with T648 and T656.  Can you fix it?

**Comment by StevenClontz** [comment]
Corrected at e76197724f22a39e162016b421fcb211b37e5e2f

**Pull request merged**